.github/workflows/release_builds.yml

```diff
@@ -417,6 +426,13 @@ jobs:
             PLATFORMS="linux/amd64" \
             BUILD_TYPE_SUFFIX="-fips"
 
+      - name: Push joint UBI-based FIPS container manifest for all platforms to Open Source ECR
+        run: |
+          make push-container-manifest \
+            BUILD_TAG=${{ steps.extract_tag.outputs.tag }} \
+            PLATFORMS="linux/amd64"
+            BUILD_TYPE_SUFFIX="-ubi-fips"
```

Comment:
I don't know if better is `-ubi-fips` or `-fips-ubi`

Reply:
I used `-ubi-fips` as for shortcut of `UBI-based FIPS image`.

Reply:
I vote for `ubi-fips`